main.go

```diff
@@ -45,8 +45,9 @@ For example, 'My Process Name [pid: 123] [Annotation]' with -pidTag=123:Annotati
 )
 
 const (
-	kSample              string = "sample"
-	kInstrumentsDeepCopy string = "instruments"
+	kSample               string = "sample"
+	kInstrumentsDeepCopy  string = "instruments"
+	kInstrumentsCollapsed string = "collapsed"
```

Comment:
Add information to the formatHelp constant above.

Reply:
Done.